apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModal.constants.ts

```diff
@@ -192,23 +192,7 @@ using ( realtime.messages.extension = 'broadcast' );`.trim(),
       command: 'SELECT',
       roles: ['authenticated'],
     },
-    {
-      id: 'policy-broadcast-2',
-      preview: false,
-      templateName: 'Allow pushing broadcasts for authenticated users only',
-      description: 'This policy allows pushing broadcasts for authenticated users only.',
-      statement: `
-create policy "Allow pushing broadcasts for authenticated users only"
-ON realtime.messages for update
-TO authenticated
-using ( realtime.messages.extension = 'broadcast' )
-with check ( realtime.messages.extension = 'broadcast' );`.trim(),
-      name: 'Allow pushing broadcasts for authenticated users only',
-      definition: "realtime.messages.extension = 'broadcast'",
-      check: "realtime.messages.extension = 'broadcast'",
-      command: 'UPDATE',
-      roles: ['authenticated'],
-    },
```

Comment:
@saltcod we still want the policies, just not the `UPDATE` ones. Can you just replace these `UPDATE` policies with `INSERT` ones?

Reply:
gotcha gotcha